features/step_definitions/routability.js

```diff
@@ -29,8 +29,13 @@ module.exports = function () {
                                     result[direction].status.toString() : '';
                                 break;
                             case /^\d+s/.test(want):
+                                outputRow[direction] = result[direction].time ?
+                                    result[direction].time.toString()+'s' : '';
                                 break;
                             case /^\d+ km\/h/.test(want):
+                                outputRow[direction] = !isNaN(result[direction].speed) ?
+                                    result[direction].speed.toString()+' km/h' :
+                                    result[direction].speed.toString() || '';
```

Comment:
speed can come back as the value `diff` in the `bothw` field, when `forw`/`backw` differ.

I was seeing that without appending `km/h` just the number is returned in the output table. I can double check that this isn't needed though.